src/components/InfoBox/InfoBox.tsx

```diff
@@ -7,7 +7,7 @@ interface Props {
   w?: number;
 }
 
-export function InfoBox({ text, w = 220 }: Props) {
+export function InfoBox({ text, w = 320 }: Props) {
   return (
     <Tooltip label={text} multiline maw={w} offset={{ mainAxis: 5, crossAxis: 100 }}>
```

Comment:
Why use maw and not w? Seems they use w in the offical docs https://mantine.dev/core/tooltip/#multiline

Reply:
To look nicer for short texts. 

With `maw`,  i.e. max width:
![image](https://github.com/user-attachments/assets/a5dd04a9-3593-4785-9cb6-f8a839a108ed)

VS with `w`, i.e. a fixed width
![image](https://github.com/user-attachments/assets/269cd1b7-860d-4f43-ae91-514cc3f074f6)